chain/actors/actor_miner.go

```diff
@@ -621,7 +512,7 @@ func (sma StorageMinerActor) SubmitFallbackPoSt(act *types.Actor, vmctx types.VM
 	}
 
 	if ok, lerr := sectorbuilder.VerifyFallbackPost(vmctx.Context(), mi.SectorSize,
-		sectorbuilder.NewSortedPublicSectorInfo(sectorInfos), seed[:], params.Proof, candidates, proverID, 0); !ok || lerr != nil { // TODO: FORK - set faults to len(faults)
+		sectorbuilder.NewSortedPublicSectorInfo(sectorInfos), seed[:], params.Proof, candidates, proverID, len(faults)); !ok || lerr != nil {
```

Comment:
TODO: convince myself that this is correct

Reply:
(probably is, we already pass that in GenerateFPpost)

Reply:
It isn't really correct. The correct number is faults in proving set (I think).
https://gist.github.com/Kubuxu/cca97d04cbb57c6e4bed62a18302d25f